tools/explorer/explorer.py

```diff
@@ -11,54 +11,46 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Main module for DBO explorer."""
+import pyfiglet
 import sys
 
+from lib import explorer_handler
+from lib import parse_input
 from lib import arg_parser
-from lib import explorer
-from lib import model
-import pyfiglet
-
 
 def main(parsed_args):
   """Main method for DBO explorer."""
   figlet_out = pyfiglet.figlet_format('DBO Explorer', font='digital')
   print(figlet_out)
   print('Starting DBO explorer...')
 
-  my_ontology = explorer.Build(parsed_args.modified_types_filepath)
+  my_ontology = explorer_handler.Build(parsed_args.modified_types_filepath)
```

Comment:
nit: modified_ontology?

Reply:
or main_ontology?

Reply:
By default if there is not modified ontology path, then the explorer will build the main ontology.

Reply:
I see, use `ontology` ? my_ontology refers to something specific ?